de.fu_berlin.inf.dpp.ui.frontend/html/webpack.config.test.js

```diff
@@ -28,6 +28,16 @@ module.exports = {
         test: /\.css$/,
         use: 'null-loader',
       },
+      {
+        test: /\.(png|jp(e*)g)$/,
+        use: [{
+          loader: 'url-loader',
+          options: {
+            limit: 8000, // Convert images < 8kb to base64 strings
+            name: 'images/[hash]-[name].[ext]'
+          }
+        }]
+      }
```

Comment:
keep in mind, [mocha-webpack](https://www.npmjs.com/package/mocha-webpack#which-version-works-with-mocha-webpack) only supports webpack up to version 3 and is no longer actively maintained, at some point we should switch to another solution.

Reply:
Thanks for the hint. I think it is necessary to update the whole build infrastructure and several packages.